site/js/main.js

```diff
@@ -0,0 +1,418 @@
+// Import all required modules
+import i18next from '../../node_modules/i18next/dist/esm/i18next.bundled.js';
+import { resources, detectLanguage, getUserSelectTranslateHTMLCode, changeLanguage } from './i18n-resources.js';
+import { Evaluate, EX, josm, toggle, dateAtWeek, newValue } from './helpers.js';
+import { YoHoursChecker } from './yohours_model.js';
+
+// Access opening_hours from global (loaded via UMD script)
+const opening_hours = window.opening_hours;
+
+// Make opening_hours available globally (needed by helpers.js)
+window.opening_hours = opening_hours;
```

Comment:
I have no idea what this is, but why not `export`ing that variable (and import in helper.js)?

Reply:
Oh crap, that was a mess from the ESM migration. I just cleaned it up. I use opening_hours here as UMD since suncalc is still included and I couldn't get that to work with ESM.